app/src/forms/form/exportService.js

```diff
@@ -89,7 +90,7 @@ const service = {
     return await flattenComponents(schema.components);
   },
 
-  _buildCsvHeaders: async (form, data, version, fields) => {
+  _buildCsvHeaders: async (form, data, version, fields, singleRow = false) => {
     /**
      * get column order to match field order in form design
      * object key order is not preserved when submission JSON is saved to jsonb field type in postgres.
```

Comment:
Hi Sergey - I noticed this comment about matching the field order to the form design. However, when I export for "WALTER - TEST 1" (added you as owner) the columns are not in the same order as the form. Do you think this will matter to the users?

Reply:
hi Walter, good point, I'll see if npm package I used handles sorting

Reply:
Or maybe it's the Object.keys() that is losing the order? Tricky!